test/battle/move_effect/telekinesis.c

```diff
@@ -0,0 +1,70 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_TELEKINESIS].effect == EFFECT_TELEKINESIS);
+}
+
+SINGLE_BATTLE_TEST("Telekinesis makes the target unable to avoid any attacks made against it")
```

Comment:
Doesn't this test need a random check since it would always succeed in a test environment regardless of accuracy?

Reply:
hit:false makes the move always miss. If you run the the test without the fix, it fails.